docs/_includes/content.md

```diff
@@ -1213,20 +1213,151 @@ We run task **3** the output would be:
 [cargo-make] INFO - Build Done  in 0 seconds.
 ```
 
+
 <a name="usage-env"></a>
 ### Environment Variables
-cargo-make enables you to defined environment variables in several ways.<br>
-Environment variables can later be used in commands, scripts, conditions, functions and more, so it is important to have a powerful way to define them for your build.
+`cargo-make` enabled the definition of environmental variables in several ways, which can later be accessed throughout task execution.
+
+Because environmental variables play a significant role in `cargo-make`, it provides multiple declarative ways to provide them at different levels of granularity.
 
+* [Declaration](#env-declaration)
+* [Note about Ordering](#env-note-about-ordering)
 * [Global Configuration](#usage-env-config)
 * [Task](#usage-env-task)
 * [Command Line](#usage-env-cli)
 * [Env File](#usage-env-file)
 * [Global](#usage-env-global)
 
+<a name="env-declaration"></a>
+#### Declaration
+
+There are multiple ways to declare environmental variables, all of which are suited for specific suitcases.
+
+##### Simple
+
+The most ordinary one is the definition of a simple `KEY=Value` pair, which is reminiscent of tools like [dotenv](https://www.npmjs.com/package/dotenv) and [bash scripts](https://www.gnu.org/software/bash/). Values can use other variables as values, which are interpolated at runtime, using the `${variable}` syntax.
+
+> **Note:** The `$variable` syntax is currently not supported!
+
+> **Note:** Extended interpolation like `${variable:-default}` is currently unsupported!
+
+```toml
+STRING = "value"
+RUST_BACKTRACE = 1
+BOOL_VALUE = true
+COMPOSITE = "${BOOL_VALUE} ${RUST_BACKTRACE}"
+```
+
+##### List
+
+`cargo-make` also supports lists, which are joined using `;` at runtime.
+
+```toml
+LIST_VALUE = [ "VALUE1", "VALUE2", "VALUE3" ]
+```
+
+##### Script
+
+`cargo-make` supports the use of simple scripts. The output of the said script will then determine the value of the environmental variable.
+
+The script's object has two additional arguments: `multiline` and `depends_on`. If `multiple` is set to `true`, the supplied script will be evaluated as a script with multiple lines. `depends_on` is a list of environmental variables this script depends on, which is taken into account during reordering if unset `cargo-make` will try to guess the variables used during reordering.
+
+> **Note:** This uses `sh` for script evaluation ([spec](https://pubs.opengroup.org/onlinepubs/9699919799/)), other runners like `duckscript`, `rust`, etc. are **not** supported.
```

Comment:
instead of:
This uses `sh` for script evaluation ([spec](https://pubs.opengroup.org/onlinepubs/9699919799/))

please write:
This uses the default OS command runner (cmd for windows, sh for the rest)